src/components/Chat/ChatExitToast/ChatExitToast.style.ts

```diff
@@ -0,0 +1,24 @@
+import styled from "@emotion/styled";
+import { pxToRem } from "~/utils";
+import { flex } from "~/styles/utils/flex";
+import type { ButtonProps } from "../ChatToastCommon.style";
+
+export const SubText = styled.div`
+  font-size: ${pxToRem(8)};
+  color: #727272;
+`;
+
+export const ButtonWrap = styled.div`
+  margin-top: ${pxToRem(17)};
+  ${flex({ justifyContent: "center" })}
+`;
+
+export const ExitButton = styled.button<ButtonProps>`
+  width: ${pxToRem(75)};
+  height: ${pxToRem(25)};
+
+  color: ${({ color }) => color || "#fff"};
```

Comment:
```suggestion
  color: ${({ color = "#fff" }) => color};
```

이렇게 기본 값을 설정할 수도 있습니다~!